src/installers/GDWeaveInstaller.ts

```diff
@@ -0,0 +1,82 @@
+import { InstallArgs, PackageInstaller } from "./PackageInstaller";
+import path from "path";
+import FsProvider from "../providers/generic/file/FsProvider";
+import { addToStateFile } from "./InstallRuleInstaller";
+import FileUtils from "../utils/FileUtils";
+
+const basePackageFiles = ["manifest.json", "readme.md", "icon.png"];
+
+// Adapted from the BepInEx & Lovely installers
+export class GDWeaveInstaller implements PackageInstaller {
```

Comment:
I've started on this, but I'm unsure how to handle the uninstall. ReturnOfModding deletes everything that isn't a folder and isn't `mods.yml` - I'll just copy this behavior for now, but let me know if I should do it differently.

Reply:
ReturnOfModding author/community had clear ideas how they wanted things to work. For GDWeave, it's better to keep things as simple as possible If there's no reason to do otherwise. My comment was a bit misleading in that sense, sorry.

Anyway, there's now this discrepancy between the install and uninstall methods: install copies everything, but uninstall only removes `winmm.dll` and `GDWeave/core`. If you're fairly certain these two will suffice for GDWeave for now, I'd suggest the installation also only copies these two and ignores everything else.